src/Libraries/CoreNodesUI/Input/BoolSelector.cs

```diff
@@ -23,6 +23,17 @@ protected override bool DeserializeValue(string val)
             }
         }
 
+        protected override bool UpdateValueCore(string name, string value)
+        {
+            if (name == "Value")
+            {
+                Value = DeserializeValue(value);
+                return true;
+            }
+
+            return base.UpdateValueCore(name, value);
+        }
+
```

Comment:
Hi @marimano, this looks reasonable but I don't see it in the latest master, is this being called only from the web server when user toggles `True` or `False` on the client?